tests/validators/tests.py

```diff
@@ -205,6 +205,11 @@
 
     (URLValidator(EXTENDED_SCHEMES), 'file://localhost/path', None),
     (URLValidator(EXTENDED_SCHEMES), 'git://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'ssh://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'git+ssh://git@github.com/example/hg-git.git', None),
+
+    # Empty schemes should not be accepted
+    (URLValidator(schemes=['']), '://www.example.com', ValidationError),
```

Comment:
Okay, I'll drop that point, however, it seems odd to me to reject an empty scheme even if someone specifies `schemes=['']` (which seems unlikely anyway). I don't know that rejecting this case is important.